.github/workflows/ci.yml

```
@@ -11,7 +11,8 @@ on:
   pull_request:
 
 jobs:
-  test:
+  test-golang:
+    name: Test GoLang
```

You can probably simplify to "go", instead of golang (as by context most developers would understand its the language). 
```suggestion
  test-go:
    name: Test Go
```